src/Microsoft.Framework.Notification/ReflectionNotifierMethodAdapter.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+// We only need ReflectionNotifierMethodAdapter for netcore50 or any case where we can't use Reflection.Emit.
+//
+// However to test it, we compile it into each test assembly. This lets us write tests without a separate
+// flavor of tests for netcore50.
```

Comment:
This seemed like the best approach to make sure that we're testing this. We don't have any existing infrastructure to test `netcore50` directly.